models.py

```diff
@@ -1,11 +1,24 @@
 from tortoise.models import Model
 from tortoise import fields
 
+class User(Model):
+	email = fields.charField(max_length=80)
+	password = fields.charField(max_length=80)
```

Comment:
I think we can do better here than storing the password in plain text. It's bad practice we don't want to spread.

You may want to take a look at [bcrypt], also this SO thread: [How to compare plain text password to hashed password using bcrypt](https://stackoverflow.com/questions/9559549/how-to-compare-plain-text-password-to-hashed-password-using-bcrypt). What we want to do I think is to have a `hashed_password` field instead of `password`.

If you don't feel comfortable tackling this I can take a look later on. Also it doesn't seem like this PR includes login verification or actually does anything with the password, so we could say that we only require the email of the user to add a new post (although that's very poor practice, too…).

Reply:
yes bcrypt is good but that is the model side. We can use bcrypt on registering and retrieving the user from our database

Reply:
I agree, password encryption is not really the database's concern, but it should be implemented at the model level.

Here's what I thought about, based on what Django does:

```python
import bcrypt
from tortoise.models import Model
from tortoise import fields

class User(Model):
    email = fields.CharField(max_length=80)
    password_hash = fields.CharField(max_length=300)

    async def create_user(self, *, password, **kwargs):
        kwargs["password_hash"] = bcrypt.hashpw(password, bcrypt.gensalt())
        return await super().create(**kwargs)
```

Then we'd have to have a login utility to compare a given password to a user's hash. We can do it the crude way, although I think it would be better to use [Starlette's auth backend abstraction](https://www.starlette.io/authentication) at some point.

Reply:
will check it